fs/fcb/src/fcb.c

```diff
@@ -110,6 +110,12 @@ fcb_free_sector_cnt(struct fcb *fcb)
 {
     int i;
     struct flash_area *fa;
+    int rc = 0;
+
+    rc = os_mutex_pend(&fcb->f_mtx, OS_WAIT_FOREVER);
+    if (rc && rc != OS_NOT_STARTED) {
+        return FCB_ERR_ARGS;
```

Comment:
Would be good to note in header file that negative value means error (or make cnt output parameter similar to suggestion in fcb_is_empty())   (either is fine for me)

Reply:
Will update the header file with the description.